t/selenium.js

```diff
@@ -794,6 +810,18 @@ async function runCommands(commands, t) {
             t.comment(timePrefix(
               `${title}: took ${elapsedTime} seconds`,
             ));
+
+            if (!didDatabaseChange) {
+              const finishTupStats = await getSeleniumDbTupStats();
+              didDatabaseChange = (
+                finishTupStats.tup_inserted > startTupStats.tup_inserted ||
+                finishTupStats.tup_updated > startTupStats.tup_updated ||
+                finishTupStats.tup_deleted > startTupStats.tup_deleted
+              );
+            }
+
+            // eslint-disable-next-line require-atomic-updates
+            shouldCleanSeleniumDb = didDatabaseChange;
```

Comment:
https://eslint.org/docs/rules/require-atomic-updates says this can cause race conditions, is this one safe then?

Reply:
It looks like a false positive to me, since the tests never run concurrently with each other. Added a comment.